python/taichi/lang/matrix.py

```diff
@@ -548,6 +548,11 @@ def norm(self, l=2, eps=0):
         assert l == 2
         return impl.sqrt(self.norm_sqr() + eps)
 
+    @taichi_scope
+    def norm_inv(self, l=2, eps=0):
+        assert l == 2
+        return impl.rsqrt(self.norm_sqr() + eps)
+
     @taichi_scope
     def norm_sqr(self):
```

Comment:
Probably in a future PR: let's deprecate `norm_sqr` and use `norm_square` instead. Some people will think of `sqr` as *square root*.

Reply:
OK, on my way, but what's the sqrt of norm... may be `sqrt(sqrt(dot(v, v)))`...?